ecli/ecli-rs/src/oci/wasm/mod.rs

```diff
@@ -31,7 +31,7 @@ pub fn parse_img_url(url: &str) -> EcliResult<(Client, RegistryAuth, Reference,
             RegistryAuth::Basic(username, password)
         }
         Err(err) => {
-            if let EcliError::LoginInfoNotFoundError(_) = err {
+            if matches!(err, EcliError::LoginInfoNotFoundError(_)) {
```

Comment:
Not sure that this change is necessary, but on deepsource.

Reply:
Well, maybe the next time we can disable b some unnecessary linters...

Reply:
And do I need squash for these commits?